### PR TITLE
Move failed ceremony dirs into a defunct dir

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

We can keep history, or just nuke'ing them is fine with me.
This avoids confusion when signers are trying to sign.

### CODE REVIEW

**Review by Hayden-IO — DISMISSED** [review]

**Comment by asraa** [comment]
validation CI detects ceremony's, will fix some of htis

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ jobs:
       - name: verify staged ceremony changes
         run: |
           set -euo pipefail
-          export CHANGED=$(git diff --name-only HEAD^ HEAD ceremony/ | grep ".*staged.*" | head -1)
+          export CHANGED=$(git diff --name-only HEAD^ HEAD ceremony/20* | grep ".*staged.*" | head -1)
```
Comment (Hayden-IO) [inline .github/workflows/validate.yml:58]: what happens in the year 3000

**Review by Hayden-IO — DISMISSED** [review]

File: .github/workflows/validate.yml (hunk 1 above)
Comment (asraa) [reply .github/workflows/validate.yml:58]: :D we celebrate our 980th anniversary with a potential outage

**Review by joshuagl — COMMENTED** [review]
Seems fine to me. It might be worth putting a README in the top-level ceremony/defunct directory so that folks can better understand what this is?

**Comment by asraa** [comment]
Nice! Added docs in the orchestration README and the top-level

**Review by joshuagl — APPROVED** [review]
Thanks!

**Pull request merged**